File: python/nivel 4 trampas/22.py
"""
Ejercicio 22
Contar cuántas veces aparece cada letra en un texto
"""

texto = input("ingrese un texto: ").lower()

conteo = {}

for letra in texto:
  if letra == " ":
    continue

  if letra in conteo:
    conteo[letra] += 1
  else:
    conteo[letra] = 1

print("Conteo de Letras")
for letra, cantitad in conteo.items():
  print(f"{letra} -> {cantitad}")

File: javascript/nivel 4 trampas/22.js
/*
Ejercicio 22
Contar cuántas veces aparece cada letra en un texto
*/
let texto = prompt("Ingresa un texto");
texto = texto.toLowerCase();

let conteo = {}

for (let letra of texto) {
  if(letra !== " "){
    if(conteo[letra]){
      conteo[letra]++;
    }else{
      conteo[letra] = 1;
    }
  }
}

console.log("Conteo de Letras");
for(let letra in conteo){
  console.log(letra + " -> " + conteo[letra]);
}